h/services/search_index/_queue.py

```diff
@@ -218,38 +125,13 @@ def sync(self, limit):
                 counts[Queue.Result.COMPLETED_TAG_TOTAL.format(tag=job.tag)].add(job.id)
                 counts[Queue.Result.COMPLETED_TOTAL].add(job.id)
 
-        for job in job_complete:
-            self._db.delete(job)
+        self._queue_service.delete_jobs(job_complete)
 
         if annotation_ids_to_sync:
             self._batch_indexer.index(list(annotation_ids_to_sync))
 
         return {key: len(value) for key, value in counts.items()}
 
-    def _get_jobs_from_queue(self, limit):
-        return (
-            self._job_query()
-            .order_by(Job.priority, Job.enqueued_at)
-            .limit(limit)
-            .with_for_update(skip_locked=True)
-            .all()
-        )
-
-    def _job_query(self, tags=None, hide_scheduled=True):
-        now = datetime.utcnow()
-
-        query = self._db.query(Job).filter(
-            Job.name == "sync_annotation", Job.expires_at >= now
-        )
-
-        if hide_scheduled:  # pragma: no cover
```

Comment:
We never filter by these. Without a comment is weird to include them anyway and no-cover.